src/main/java/org/breedinginsight/brapps/importer/model/workflow/ExperimentWorkflow.java

```diff
@@ -0,0 +1,6 @@
+package org.breedinginsight.brapps.importer.model.workflow;
+
+@FunctionalInterface
+public interface ExperimentWorkflow extends Workflow {
```

Comment:
Is it necessary to have to levels of Workflow interface implementations? Here and ExperimentWorkflowNavigator

Reply:
I implemented this for ExperimentImportService, so you can see the details there. The same can be done for Germplasm and SampleSubmission once WorkflowNavigators are created.